portability-worker/src/main/java/org/dataportabilityproject/worker/PortabilityTransferCopier.java

```diff
@@ -0,0 +1,85 @@
+package org.dataportabilityproject.worker;
+
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.dataportabilityproject.spi.transfer.TransferCopier;
+import org.dataportabilityproject.spi.transfer.provider.ExportResult;
+import org.dataportabilityproject.spi.transfer.provider.Exporter;
+import org.dataportabilityproject.spi.transfer.provider.Importer;
+import org.dataportabilityproject.spi.transfer.provider.TransferServiceProviderRegistry;
+import org.dataportabilityproject.spi.transfer.types.ContinuationData;
+import org.dataportabilityproject.spi.transfer.types.ExportInformation;
+import org.dataportabilityproject.types.transfer.PortableType;
+import org.dataportabilityproject.types.transfer.auth.AuthData;
+import org.dataportabilityproject.types.transfer.models.ContainerResource;
+import org.dataportabilityproject.types.transfer.models.DataModel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PortabilityTransferCopier implements TransferCopier {
```

Comment:
Add java doc and what if we rename to something to the effect of in memory copier (since we may break copying up later to take part between two different binaries)

Reply:
Done.